build/digests.json

```diff
@@ -32,14 +32,14 @@
    "_total_contributions_from_totals": 35003.0
   },
   "oakland-2024": {
-   "_total_contributions_by_source_with_unitemized_from_elections": 17045.0,
-   "_total_contributions_by_source_with_unitemized_from_tickets": 17045.0,
-   "_total_contributions_by_source_with_unitemized_from_totals": 230239.05,
-   "_total_contributions_by_type_with_loans_from_elections": 17045.0,
-   "_total_contributions_by_type_with_loans_from_tickets": 17045.0,
-   "_total_contributions_by_type_with_loans_from_totals": 230239.05,
-   "_total_contributions_from_elections": 17045.0,
-   "_total_contributions_from_tickets": 17045.0,
+   "_total_contributions_by_source_with_unitemized_from_elections": 220239.05,
```

Comment:
Why are these so different?

Reply:
I don't know right now.  I've just made the hidden problems visible and hopefully I'll find some time to track down those hidden problems, using the auto-deployed build directory as a guide.  The PR for the csvkit upgrade is where I'm starting to do this, so hopefully we'll find some time to discuss the differences surfaced there.

Reply:
This change does not effect any processing or data checking, just does autodeploy. How could it possibly reveal hidden problems?